src/components/Terminal.tsx

```diff
@@ -0,0 +1,24 @@
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter';
+import cpp from 'react-syntax-highlighter/dist/esm/languages/hljs/cpp';
+import atomOneLightCustom from '../styles/Terminal/atom-one-light-custom';
+import '../styles/Terminal/Terminal.scss';
+
+export interface TerminalProps {
+  code: string;
+}
+
+SyntaxHighlighter.registerLanguage('cpp', cpp);
+
+export default function Terminal(props: TerminalProps): JSX.Element {
+  return (
+    <div className="terminal">
+      <SyntaxHighlighter
+        language="cpp"
+        style={atomOneLightCustom} // this error doesn't affect the custom theme, will fix later
```

Comment:
What are you trying to do here out of curiousity? It looks fine right now, so I think it's fine to leave until later.

Reply:
SyntaxHighlighter comes with a bunch of preset themes which can be set using the style attribute. I wanted to change the background atomOneLight to match the Figma design so I had to create a new theme to do so. The error on line 17 has to do with a bad overload of the SyntaxHighlighter constructor because atomOneLightCustom doesn't have the correct instantiations in ./src/styles/Terminal/index.d.ts to be used in SyntaxHighlighter component. It doesn't seem to have any affects but should probably be cleaned up.